```
[...]/src/libsolidity/lsp/DocumentHoverHandler.cpp:66:27: error: chosen constructor is explicit in copy-initialization
        MarkdownBuilder markdown{};
                                 ^
[...]/sysroot/usr/include/c++/v1/sstream:884:14: note: explicit constructor declared here
    explicit basic_stringstream(ios_base::openmode __wch = ios_base::in | ios_base::out);
             ^
[...]/src/libsolidity/lsp/DocumentHoverHandler.cpp:38:15: note: in implicit initialization of field 'result' with omitted initializer
        stringstream result;
                     ^
```

--- a/libsolidity/lsp/DocumentHoverHandler.cpp
+++ b/libsolidity/lsp/DocumentHoverHandler.cpp
@@ -32,7 +32,7 @@
 
 struct MarkdownBuilder
 {
-	std::stringstream result;
+	std::stringstream result{};
 
 	MarkdownBuilder& solidityCode(std::string const& _code)
 	{
